lib/react-d3/extractData.js

```diff
@@ -27,14 +27,19 @@ module.exports = (nodes) => {
     //HTML tag name ...div, g, circle, etc...
     output.tag = obj.localName;
     const sameTagIndex = i + document.getElementsByTagName(obj.localName).length;
+    let children = [];
+    const childNodes = obj.childNodes;
+    for(let i=0; i < obj.childNodes.length; i++) {
+      if(childNodes[i].tagName) children.push(childNodes[i]);
```

Comment:
In IE case, childNodes method can get value.

e.x.)
``` <text>B</text>```
childNodes method can get " B".

So we need to check tagName to except text value.